sparkctl/cmd/client.go

```diff
@@ -24,9 +24,22 @@ import (
 
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta1"
 	crdclientset "github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/client/clientset/versioned"
+	"os"
```

Comment:
This should be at the top of the import list with other built-in packages.

Reply:
Fixed